deps/build.jl

```diff
@@ -7,15 +7,40 @@ if !isfile(config_toml)
     touch(config_toml)
 end
 
+# This file contains code that runs at three different occasions:
```

Comment:
This is a great comment!

Reply:
I also spent a great amount of time figuring this out.